priv/js/app.js

```diff
@@ -32,6 +36,16 @@ window.addEventListener("keydown", e => {
   }
 })
 
+async function isValidSite(siteName) {
+
+  let similarSites = await Promise.resolve(fetchSimilarSites(siteName))
```

Comment:
the `Promise.resolve` wrapper isn't needed here since `fetchSimilarSites` already returns a Promise (it's an `async function` and those always return a Promise). Can just be:
```js
let similarSites = await fetchSimilarSites(siteName)
```